wire-benchmarks/src/jmh/java/com/squareup/wire/benchmarks/EncodeBenchmark.java

```diff
@@ -17,9 +17,13 @@
 
```

Comment:
File name is bad, probably will clean things up in followup as I'll be adding messages that Google is using on their own benchmarks
https://github.com/protocolbuffers/protobuf/tree/master/benchmarks/datasets